doc/api/fs.md

```diff
@@ -2066,16 +2066,29 @@ changes:
 -->
 
 * `path` {string|Buffer|URL}
-* `mode` {integer} Not supported on Windows. **Default:** `0o777`.
+* `options` {Object|integer}
+  * `createParents` {boolean} **Default:** `false`
+  * `mode` {integer} Not supported on Windows. **Default:** `0o777`.
 * `callback` {Function}
   * `err` {Error}
 
 Asynchronously creates a directory. No arguments other than a possible exception
 are given to the completion callback.
 
+The optional `options` argument can be an integer specifying mode (permission
+  and sticky bits), or an object with a mode property and a parent property
```

Comment:
1. Unneeded leading space.
2. `with a mode property and a parent property` -> `` with `mode` property and `createParents` property ``?